console.rst

```diff
@@ -79,18 +79,37 @@ Configuring the Command
 You can optionally define a description, help message and the
 :doc:`input options and arguments </console/input>`::
 
-    // ...
-    protected function configure()
-    {
-        $this
-            // the short description shown while running "php bin/console list"
-            ->setDescription('Creates a new user.')
+.. versionadded:: 5.3
 
-            // the full command description shown when running the command with
-            // the "--help" option
-            ->setHelp('This command allows you to create a user...')
-        ;
-    }
+    The ability to use PHP attributes to configure commands was introduced in
+    Symfony 5.3.
+
+.. configuration-block::
+
+    .. code-block:: php-attributes
+
+            // ...
+            #[ConsoleCommand(
+                name: 'app:create-user',
+                description: 'Creates a new user.'
+                hidden: false,
+                aliases: [],
```

Comment:
It would be interesting how to set the help

Reply:
Currently, help isn't supported in attributes and the use of `setHelp()` is necessary. 
Should I add it in addition to the attributes?